ansible/group_vars/tag_Environment_alpha

```diff
@@ -55,26 +58,17 @@ register_groups:
     - field
 
   multi:
-    - academy-school-eng
-    - diocese
     - government-organisation
     - government-service
     - internal-drainage-board
-    - la-maintained-school-eng
     - local-authority-eng
     - local-authority-sct
     - local-authority-type
     - notifiable-animal-disease
     - notifiable-animal-disease-confirmation-category
     - notifiable-animal-disease-investigation-category
-    - prison
-    - religious-character
+    - principal-local-authority
+    - prison-estate
     - registration-district
-    - school-admissions-policy
-    - school-authority-eng
     - school-eng
-    - school-gender
-    - school-phase
-    - school-tag
```

Comment:
I take it all these school related things are related to moving everything into `school-eng`?